src/main/java/kr/pickple/back/game/domain/Game.java

```diff
@@ -131,22 +136,38 @@ private Game(
         this.addressDepth2 = addressDepth2;
     }
 
-    public void addGamePosition(final GamePosition gamePosition) {
-        gamePosition.setGame(this);
-    }
-
     public void addGamePositions(final List<String> positions) {
-        final List<GamePosition> gamePositions = positions
-                .stream()
+        positions.stream()
                 .distinct()
                 .map(position -> GamePosition.builder()
                         .position(Position.from(position))
                         .game(this)
                         .build())
-                .toList();
+                .forEach(this::addGamePosition);
+    }
+
+    private void addGamePosition(final GamePosition gamePosition) {
+        gamePosition.updateGame(this);
+    }
+
+    public void addGameMember(final Member member) {
+        validateIsAlreadyRegisteredGameMember(member);
+
+        final GameMember gameMember = GameMember.builder()
+                .member(member)
+                .game(this)
+                .build();
+
+        gameMembers.add(gameMember);
+    }
+
+    private void validateIsAlreadyRegisteredGameMember(final Member member) {
+        final boolean isAlreadyRegisteredGameMember = gameMembers.stream()
+                .anyMatch(gameMember -> member == gameMember.getMember());
```

Comment:
`p2`

- 해당 로직은 이미 해당 경기에 게임 멤버로 등록되어있는지 체크하는 검증 로직인 것 같습니다.
- 현재 gameMembers를 순회하며 gameMember가 매개변수로 들어온 Member와 동일한지 체크하는 로직 같은데 현재 객체 비교로 되어있어 동일한 값을 가진 인스턴스라도 인스턴스의 주소를 비교하기 때문에 다르다고 판별할 수 있을 것 같습니다.
- 아래와 같이 바꾸면 안전한 검증이 되지 않을까 싶은데 영주님은 어떻게 생각하시나요?!
```suggestion
                .anyMatch(gameMember -> member.getId() == gameMember.getMember().getId());
```

Reply:
반영 : [b3be817](https://github.com/Java-and-Script/pickple-back/pull/63/commits/b3be8176203cacdfe9eca4621a28a608c084b558)
- 반영 완료!